rapidtide/workflows/rapidtide_parser.py

```diff
@@ -908,279 +1202,341 @@ def process_args(inputargs=None):
+    )
 
-    if args['debug']:
-        print()
-        print('before postprocessing')
-        print(args)
+    LGR.debug("\nbefore postprocessing:\n{}".format(args))
 
     # some tunable parameters for internal debugging
-    args['dodemean'] = True
-    args['edgebufferfrac'] = 0.0  # what fraction of the correlation window to avoid on either end when fitting
-    args['enforcethresh'] = True  # only do fits in voxels that exceed threshhold
-    args['lagmod'] = 1000.0  # if set to the location of the first autocorrelation sidelobe, this will fold back sidelobes
-    args['lthreshval'] = 0.0  # zero out peaks with correlations lower than this value
-    args['uthreshval'] = 1.0  # zero out peaks with correlations higher than this value
-    args['absmaxsigma'] = 10000.0  # width of the reference autocorrelation function
-    args['absminsigma'] = 0.25  # width of the reference autocorrelation function
+    args["dodemean"] = True
+    # what fraction of the correlation window to avoid on either end when
+    # fitting
+    args["edgebufferfrac"] = 0.0
+    # only do fits in voxels that exceed threshhold
+    args["enforcethresh"] = True
+    # if set to the location of the first autocorrelation sidelobe,
+    # this will fold back sidelobes
+    args["lagmod"] = 1000.0
+    # zero out peaks with correlations lower than this value
+    args["lthreshval"] = 0.0
+    # zero out peaks with correlations higher than this value
+    args["uthreshval"] = 1.0
+    # width of the reference autocorrelation function
+    args["absmaxsigma"] = 10000.0
+    # width of the reference autocorrelation function
+    args["absminsigma"] = 0.25
 
     # correlation fitting
-    args['hardlimit'] = True  # Peak value must be within specified range.  If false, allow max outside if maximum
-                              # correlation value is that one end of the range.
-    args['searchfrac'] = 0.5  # The fraction of the main peak over which points are included in the peak
-    args['mp_chunksize'] = 50000
+    # Peak value must be within specified range.
+    # If false, allow max outside if maximum
+    # correlation value is that one end of the range.
+    args["hardlimit"] = True
+    # The fraction of the main peak over which points are included in the peak
+    args["searchfrac"] = 0.5
+    args["mp_chunksize"] = 50000
 
     # significance estimation
-    args['sighistlen'] = 1000
-    args['dosighistfit'] = True
+    args["sighistlen"] = 1000
+    args["dosighistfit"] = True
 
     # output options
-    args['savecorrmask'] = True
-    args['savedespecklemasks'] = True
-    args['saveglmfiltered'] = True
-    args['savemotionfiltered'] = False
-    args['savecorrmask'] = True
-    args['histlen'] = 250
+    args["savecorrmask"] = True
+    args["savedespecklemasks"] = True
+    args["saveglmfiltered"] = True
+    args["savemotionfiltered"] = False
+    args["savecorrmask"] = True
```

Comment:
I think this is a duplicate of L1283.

Reply:
I don't see a line 1283 in view changes...

Reply:
I think maybe it's using the line numbers from the "before" file then. The line I'm referring to is 5 lines above the one I commented from.

Reply:
Ah, ok.  You are right.